ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -19,6 +192,401 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '67EF', source: 'ゾディアーク' }),
       response: Responses.tankBuster(),
     },
+    {
+      id: 'ZodiarkEx Kokytos',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6C60', source: 'Zodiark', capture: false }),
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'ZodiarkEx Paradeigma',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '67BF', source: 'Zodiark', capture: false }),
+      promise: async (data, _matches) => {
+        console.log('p cnt ', data.paradeigmaCounter);
+        // TODO: Since fetch already supports arrays don't bother overlay twice
+        if (data.paradeigmaCounter === 2 || data.paradeigmaCounter === 3 || data.paradeigmaCounter === 5 || data.paradeigmaCounter === 6) {
+          const python = await fetchCombatantsByBNpcID([14387]);
+          /*
+          console.log('------------');
+          for (const p of python)
+            console.log('paradeigma python: ', p);
+          console.log('------------');
+          */
+          data.activePythons = python;
+        }
+        if (data.paradeigmaCounter === 0 || data.paradeigmaCounter === 1 || data.paradeigmaCounter === 4 || data.paradeigmaCounter === 5 || data.paradeigmaCounter === 7) {
+          const quetz = await fetchCombatantsByBNpcID([14388]);
+          /*
+          console.log('------------');
+          for (const q of quetz)
+            console.log('paradeigma quetz: ', q);
+          console.log('------------');
+          */
+          data.activeQuetzs = quetz;
+        }
+      },
+      alertText: (data, _matches, output) => {
+        ++data.paradeigmaCounter;
+        if (data.paradeigmaCounter === 1)
+          return output.underQuetz!();
+      },
+      outputStrings: {
+        underQuetz: {
+          en: 'Under NW Quetzalcoatl',
+        },
+      },
+    },
+    {
+      id: 'ZodiarkEx Styx',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67F3', source: 'Zodiark', capture: false }),
+      response: Responses.aoe(),
```

Comment:
It's not always 6 stacks I think because it can extra hit somehow (when there are dead people? or am I just always nervous when things go sideways?)

Anyways I changed it to stack x6